supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc

```diff
@@ -233,7 +233,7 @@ Always use "VM Portal" and capitalize the product name.
 [discrete]
 [[vnic]]
 ==== image:images/yes.png[yes] vNIC (noun)
-*Description*: "vNIC" is an abbreviation for virtual network interface controller. Use lowercase v and uppercase NIC for the abbreviation, but all lowercase for the expansion, except at the beginning of a sentence.
+*Description*: _vNIC_ is an abbreviation for "virtual network interface controller". Use lowercase v and uppercase NIC for the abbreviation, but all lowercase for the expansion, except at the beginning of a sentence.
```

Comment:
se lowercase "v" and uppercase "NIC" for the abbreviation, but all lowercase for the expansion, except at the beginning of a sentence.